Library/Formula/envchain.rb

```diff
@@ -0,0 +1,14 @@
+class Envchain < Formula
+  homepage "https://github.com/sorah/envchain"
+  head "https://github.com/sorah/envchain.git"
+  url "https://github.com/sorah/envchain/archive/v0.2.0.tar.gz"
+  sha256 "2a863688d1e0bdc47ba8339f57c8b5e22f5002fd3ab58928766e45f23c6ca267"
+
+  def install
+    system "make", "DESTDIR=#{prefix}", "install"
+  end
+
+  test do
+    system "#{bin}/envchain 2>&1 | grep 'envchain version #{version}'"
```

Comment:
I was worried that interacting with the keychain would be cause for potential CI pain, but let me give it a shot and I'll update one way or the other. :+1: